Assets/_BuildingBlocks/Credentials/Scripts/KeyVaultCoroutineRunner.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+
+namespace Netherlands3D.Credentials
+{
+    public class KeyVaultCoroutineRunner : MonoBehaviour
+    {
+        private static KeyVaultCoroutineRunner instance;
+
+        public static KeyVaultCoroutineRunner Instance()
+        {
+            if (!instance == false) return instance;
```

Comment:
Je doet nu een dubbele negative, dat kan toch ook zo?
`if(instance) return instance;`

Reply:
Ik zal het aanpassen, deze constructie gebruikte ik soms om meer te leunen op de operator overrides van Unity; maar dit is niet een van die situaties